hashicorp/terraform-helm
# Add helm acceptance tests

This PR includes acceptance tests for the Helm chart.

By running

```shell
$ bats ./test/acceptance
```

The test will:

- deploy the operator to the `default` namespace
- create `terraformrc` and `workspacesecrets` Kubernetes secrets
- deploy a `Workspace` to the Terraform Cloud organization specified
   using the `test/module` module.
- check for outputs of the module using a pod (AKA runner)
- delete workspace, Helm installation, and secrets

These acceptance tests can be used to verify new releases of `terraform-k8s`.
